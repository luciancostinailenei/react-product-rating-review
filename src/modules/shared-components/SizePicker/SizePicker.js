import React from 'react'

const SizePicker = () => {
  return (
    <div className="size-picker">
      <span className="size-picker__title">Size: </span>

      <span className="size-picker__size size-picker__size--out-of-stock">
        37
      </span>
      <span className="size-picker__size size-picker__size--selected">38</span>
      <span className="size-picker__size">39</span>
      <span className="size-picker__size">40</span>
      <span className="size-picker__size">41</span>
      <span className="size-picker__size">42</span>
    </div>
  )
}

export default SizePicker
