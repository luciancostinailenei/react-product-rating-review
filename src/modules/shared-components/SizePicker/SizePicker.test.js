import React from 'react'
import toJson, { createSerializer } from 'enzyme-to-json'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import './SizePicker'
import SizePicker from './SizePicker'

configure({ adapter: new Adapter() })

describe('ProductReview', () => {
  it('matches snapshot', () => {
    const SizePickerWrapper = shallow(<SizePicker />)

    expect(toJson(SizePickerWrapper)).toMatchSnapshot()
  })
})
