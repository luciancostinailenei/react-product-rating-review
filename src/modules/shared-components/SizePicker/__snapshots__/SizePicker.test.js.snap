// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductReview matches snapshot 1`] = `
<div
  className="size-picker"
>
  <span
    className="size-picker__title"
  >
    Size: 
  </span>
  <span
    className="size-picker__size size-picker__size--out-of-stock"
  >
    37
  </span>
  <span
    className="size-picker__size size-picker__size--selected"
  >
    38
  </span>
  <span
    className="size-picker__size"
  >
    39
  </span>
  <span
    className="size-picker__size"
  >
    40
  </span>
  <span
    className="size-picker__size"
  >
    41
  </span>
  <span
    className="size-picker__size"
  >
    42
  </span>
</div>
`;
